harness/benchmarks/godot-project.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Godot.NET.Sdk/3.2.3">
+<Project Sdk="Godot.NET.Sdk/3.3">
```

Comment:
This will not work as the target framework version also need to change IIRC
Once i have mono setup again, I'll push a fix to this PR